docs/environment-docs.yml

```
@@ -4,12 +4,14 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.8
+  - python=3.9
   #
   # melodies_monet deps
   - monet
   - monetio
   - netcdf4
+  - numpy<1.24
```

```suggestion
  - numpy
```
Should be ok to remove the upper bound now. https://github.com/numba/numba/issues/8464